provider/pkg/host/fake/monitor.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint:govet
```

Comment:
Out of curiosity, what linting warning are we suppressing here?

Reply:
It is due to direct use of Pulumi RPC types, which contain a lock and trigger this:
```
pkg/host/fake/monitor.go:114:9: copylocks: range var reg copies lock: github.com/pulumi/pulumi-kubernetes/provider/v4/pkg/host/fake.Registration contains github.com/pulumi/pulumi/sdk/v3/proto/go.RegisterResourceRequest contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex (govet)
```

You're right that it is too obscure. I'm going to move the `nolint` directive to be inline on each line that needs it.